net-analyzer/synscan/synscan-5.02.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools eutils
+
+DESCRIPTION="A fast asynchronous half-open TCP portscanner"
+HOMEPAGE="http://www.digit-labs.org/files/tools/synscan/"
+SRC_URI="http://www.digit-labs.org/files/tools/${PN}/releases/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="kernel_FreeBSD kernel_linux"
+
+RDEPEND="net-libs/libpcap"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	mv "$S"/configure.in "$S"/configure.ac
+	eapply_user
```

Comment:
I suggest you remote `eapply_user` and add `default` as the first line in `src_prepare`, like this:
```
rc_prepare() {
	default
	mv "$S"/configure.in "$S"/configure.ac
	eautoconf
}
```

Reply:
ok.